styles/base.css

```diff
@@ -228,92 +194,30 @@ header {
     width: 5.5rem;
 }
 
-/* Main */
-
-main {
-    flex: 1 0 auto;
-}
-
 /* Footer */
 
-.divider-line {
-    border-top: 1px solid #545454;
-    margin: 10px -16px 10px;
-}
-
-@media (min-width: 992px) {
-    .divider-line {
-        margin: 20px -30px 20px;
-    }
-
-}
-
-footer {
-    flex-shrink: 0;
-}
-
-:root {
-    --certs-div-padding-left: 0;
-}
-
-@media (min-width: 992px) {
-    :root {
-        --certs-div-padding-left: 16px;
-    }
-}
-
-footer .certs>div {
-    margin: 30px 0 0;
-    padding-left: var(--certs-div-padding-left);
-}
-
-footer .heading {
-    font-size: 1.5rem;
-    color: var(--teal);
-    margin: 0;
-}
-
-footer .contact-link {
-    font-size: 1.125rem;
-    color: var(--green);
-    margin: 0;
-}
-
-footer .address {
-    margin-top: 24px;
-}
-
 .social-media-icon {
-    opacity: 40%;
     transition: 250ms;
-    margin: 0 12px;
-    width: calc(28rem / 16);
+    width: calc(24rem / 16);
 }
 
 @media (min-width: 992px) {
     .social-media-icon {
-        margin: 0 24px;
-        width: 3rem;
```

Comment:
For the social media icon margins, I used the Bootstrap `gap` class instead:

`<div class="d-flex py-4 gap-md-4 gap-3">`

`gap-3` sets the children apart by 16px, and `gap-md-4` sets the children apart by 24px at medium/large screen sizes. Adding `d-flex` allows you to use gap classes.

More on gap: https://getbootstrap.com/docs/5.3/utilities/spacing/#gap